src/frontend/apps/standalone_site/src/style/theme.extend.ts

```diff
@@ -0,0 +1,23 @@
+import { normalizeColor } from 'grommet/utils';
+import { theme } from 'lib-common';
+
+const textLight = normalizeColor('blue-active', theme);
```

Comment:
You should be able to use the name of the color (`blue-active`) in the theme rather than normalizing it to get the HEX value